### ISSUE TITLE
Old docker image and Travis build red for a long time

### ISSUE THREAD

**Issue by bargenson** [aweber/rabbitmq-autocluster#145]
I'm currently trying to scale my RabbitMQ cluster using Docker Swarm Mode and DNS Round Robin.

I noticed that Docker image I use (the latest you deployed on Docker Hub) contains an issue that was fixed in the PR #127, middle of February. The latest Docker image was pushed more than 6 months ago.

I tried to rebuild the Docker image with the fix but I have trouble to build the project following the instructions in the Travis file. Since the PR has been merged, none of the Travis builds succeeded.

Could you push a new version of the project to Docker Hub or help me build one on my side?

Thanks a lot!

**Comment by michaelklishin** [comment]
@bargenson see [this note on project maturity and development activity](https://github.com/aweber/rabbitmq-autocluster#project-maturity-and-status).

**Comment by michaelklishin** [comment]
I don't see anything in #127 that might affect Docker. Next release of the rabbitmq/rabbitmq-autocluster fork will happen when a few more known AWS issues are investigated and resolved.

If you post more details than "I have trouble" perhaps someone can help.

**Comment by bargenson** [comment]
@michaelklishin #127 fix the DNS round robin issue I have (inet_res:gethostbyaddr/1 doesn't work, inet:gethostbyaddr/1 does). The current published Docker image (aweber/rabbitmq-autocluster) is older than the fix provided by #127.

So, I tried to build a new Docker image with what is currently in master. I used what is declare in the .travis and I get something similar to what you have here: https://travis-ci.org/aweber/rabbitmq-autocluster. Seems to be a test failure.

I can try to get the latest commit where the build was successful and merge the commit of #127 in it and see if i'm able to build.
